src/functions/template-tags/post.php

```diff
@@ -63,3 +63,32 @@ function tribe_the_content( $more_link_text = null, $strip_teaser = false, $post
 	echo tribe_get_the_content( $more_link_text, $strip_teaser, $post_id );
 }
 
+/**
+ * Wrapper for post_class function that allows us to in-memory cache
+ *
+ * @since TBD
+ *
+ * @param string|string[] $class   Space-separated string or array of class names to add to the class list.
+ * @param int|WP_Post     $post_id Optional. Post ID or post object.
```

Comment:
:no_entry_sign: Doc comment for parameter $post_id does not match actual variable name $post

<sub>hash: 62c83392736a2158eb15ab42639c73db</sub>